### PR TITLE
[RFR] Moved some things for upcoming refactor

### PR DESCRIPTION
**Pull request by psav**
Moved some things for a big refactor

### CODE REVIEW

File: cfme/middleware/server.py [hunk 1]
```diff
@@ -16,7 +16,7 @@
 from utils.db import cfmedb
 from utils.providers import get_crud, get_crud_by_name, list_providers
 from utils.varmeth import variable
-from . import LIST_TABLE_LOCATOR, pwr_btn, MiddlewareBase, download
+from . provider import LIST_TABLE_LOCATOR, pwr_btn, MiddlewareBase, download
```
Comment (RonnyPfannschmidt) [inline cfme/middleware/server.py]: the leading space looks strange

File: cfme/middleware/server_group.py [hunk 2]
```diff
@@ -11,8 +11,8 @@
 from utils.varmeth import variable
 from utils.appliance import Navigatable
 from utils.appliance.implementations.ui import navigator, CFMENavigateStep, navigate_to
-from . import LIST_TABLE_LOCATOR, MiddlewareBase, download
-from cfme.middleware.domain import MiddlewareDomain
+from . provider import LIST_TABLE_LOCATOR, MiddlewareBase, download
```
Comment (RonnyPfannschmidt) [inline cfme/middleware/server_group.py]: dito

File: cfme/middleware/server.py (hunk 1 above)
Comment (mshriver) [inline cfme/middleware/server.py]: +1

**Comment by jkandasa** [comment]
@psav Middleware changes looks good for me.

**Pull request merged**